var/spack/repos/builtin/packages/py-ipykernel/package.py

```diff
@@ -40,3 +40,7 @@ class PyIpykernel(PythonPackage):
     depends_on('py-pytest-cov', type='test')
     # depends_on('py-flaky', type='test')
     depends_on('py-nose', type='test')
+
+    def install_data(self):
```

Comment:
Just double-checking: that's after `install`, right?
(Worked for my test but I did some back & forth locally.)

Reply:
We don't normally run this phase, but you can add:
```python
phases = ['build', 'install', 'install_data']
```

Reply:
It seems to be run automatically when added :thinking:

Reply:
Hmm... that sounds like magic. I don't think we added any logic like that...